### PR TITLE
Add rake tasks for republishing/redirecting WorldLocations

### PR DESCRIPTION
**Pull request by gpeng**
Add task to republish all `WorldLocation`

This will be done when the routes have been changed to cause them to redirect to what will become the taxon pages (`/world/<slug>`)

[Trello](https://trello.com/c/6tio72jl/221-split-world-locations-from-international-delegations-in-whitehall)

This is now on top of https://github.com/alphagov/whitehall/pull/3317 so as we can test to staging. Only the last commit is part of this PR.

### CODE REVIEW

File: lib/tasks/worldwide_taxonomy.rake [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  task redirect_world_location_translations_to_en: :environment do
+    base_path_prefix = "/government/world"
+    world_locations = WorldLocation.all
+
+    world_locations.each do |world_location|
+      en_slug = world_location.slug
+      destination_base_path = File.join("", base_path_prefix, en_slug)
+      content_id = world_location.content_id
+      locales = world_location.original_available_locales - [:en]
```
Comment (bevanloon) [inline lib/tasks/worldwide_taxonomy.rake]: Is `original_available_locales` a thing? It fails on my VM in the console:

```
irb(main):008:0> WorldLocation.first.original_available_locales
  WorldLocation Load (2.0ms)  SELECT  `world_locations`.* FROM `world_locations`  ORDER BY `world_locations`.`id` ASC LIMIT 1
NoMethodError: undefined method `original_available_locales' for #<WorldLocation:0x007f43bb565c60>
```

`available_locales` works:

```
irb(main):009:0> WorldLocation.first.available_locales
  WorldLocation Load (0.5ms)  SELECT  `world_locations`.* FROM `world_locations`  ORDER BY `world_locations`.`id` ASC LIMIT 1
  WorldLocation::Translation Load (0.1ms)  SELECT `world_location_translations`.* FROM `world_location_translations` WHERE `world_location_translations`.`world_location_id` = 1
=> [:en, :dr]
```

Comment (gpeng) [inline lib/tasks/worldwide_taxonomy.rake]: Oh yeah... it isn't until the other PR gets merged. Maybe these need to be in two separate PRs 🤕
